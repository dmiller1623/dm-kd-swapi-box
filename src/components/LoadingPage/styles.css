.loading-page {
  width: 100vw;
  height: 100vh;
  background-image: url(https://steamusercontent-a.akamaihd.net/ugc/916918850613696016/E5903247129038C83FB540F2A32CEF4A388E2C72/);
  background-repeat: no-repeat;
  background-size: 100% 100%; }
