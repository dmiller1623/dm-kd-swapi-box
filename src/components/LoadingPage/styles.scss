.loading-page-people {
  width: 100vw;
  height: 100vh;
  background-image: url(https://steamusercontent-a.akamaihd.net/ugc/916918850613696016/E5903247129038C83FB540F2A32CEF4A388E2C72/);
  // background-image: url(http://i.imgur.com/tbVomBn.gif);
  background-repeat: no-repeat;
  background-size: 100% 100%;
}

.loading-page-planets {
  width: 100vw;
  height: 100vh;
  // background-image: url(https://media.giphy.com/media/NGIfqtcS81qi4/giphy.gif);
  background-image: url(https://pa1.narvii.com/6335/6ded4e2d3e8f52689fb49c0f52b7b248eab57859_hq.gif);

  background-repeat: no-repeat;
  background-size: 100% 100%;
}

.loading-page-vehicles {
  width: 100vw;
  height: 100vh;
  background-image: url(https://cdn-images-1.medium.com/max/2000/1*cgz_L67P_7WtCnMMdeCRNA.gif);
  background-repeat: no-repeat;
  background-size: 100% 100%;
}