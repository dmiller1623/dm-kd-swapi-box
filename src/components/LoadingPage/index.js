import React from 'react'
import './styles.css'

const LoadingPage = ({ cardType }) => {
  if(cardType === 'people') {
    return ( 
      <div className='loading-page-people'></div>
    )
  }  
  else if(cardType === 'planets') {
    return (
      <div className='loading-page-planets'></div>
    )
  }
  else if(cardType === 'vehicles') {
    return ( 
       <div className='loading-page-vehicles'></div>
    )
  }  
}

export default LoadingPage