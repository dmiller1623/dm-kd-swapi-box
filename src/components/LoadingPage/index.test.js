import React from 'react'
import LoadingPage from './index.js'
import { shallow } from 'enzyme'

describe('LoadingPage', () => {
  it('should match the snaphot', () => {
    let wrapper = shallow(<LoadingPage />)
    expect(wrapper).toMatchSnapshot()
  })
})