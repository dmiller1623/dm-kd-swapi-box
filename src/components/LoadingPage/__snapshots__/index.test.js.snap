// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage should match the snaphot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LoadingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): undefined,
  Symbol(enzyme.__nodes__): Array [
    undefined,
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
